code/modules/mob/living/silicon/robot/robot_vr.dm

```diff
@@ -140,6 +140,8 @@
 	if(wideborg == TRUE)
 		if(icontype == "Drake") // Why, Why can't we have normal nice things
 			icon = 'icons/mob/drakeborg/drakeborg_vr.dmi'
+		else if(icontype == "MediRaptor model V-4" || icontype == "SecRaptor model V-4"|| icontype == "SciRaptor model V-4") //Added for raptor sprites
```

Comment:
I don't really like this jank. Maybe rename them all to be "Raptor V-4" or other 'universal' name across modules to avoid snowflaking like that?

Reply:
Fine by me. Also, I've noticed that these don't have resting sprites, causing them to go invisible when they rest. I'll fix that straight away.